webcompat/static/js/lib/models/issue.js

```diff
@@ -102,9 +91,14 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var regex = /\*\*Description\*\*: ([^*]+)/;
-    var description = regex.exec(body);
-    return description != null ? description[1].slice(0, 74) : null;
+    var el = document.createElement("html");
+    el.innerHTML = body;
+    el = el.getElementsByTagName("strong");
+    if (el[5]) {
```

Comment:
This seems fragile -- what if the DOM structure changes? 

What is the 6th `strong` element? Some comments in the code would probably help me understand (and the next person who has to deal with this). Can we get the description with a CSS selector and `querySelector` rather than this kind of DOM traversal? It would also fail if the DOM changed, but it seems easier to understand. 

Another option would be to use the jQuery contains selector: https://api.jquery.com/contains-selector/

Reply:
Yeah.. I figured it wasn't the best way. :p
What do you think of this new solution?
I thought it was too simple to write a comment explaining it, but I might be wrong.
Thanks for the tip :) I don't know much about jquery.